### PR TITLE
Upgrade to PHP 8.2

### PR DESCRIPTION
**Pull request by eldadfux**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -53,14 +54,14 @@ RUN \
...
-    make && make install && \
-    cd ..;
+# RUN cd /tmp && \
+#     apk add boost-dev && \
+#     git clone --depth 1 https://github.com/swoole/yasd && \
+#     cd yasd && \
+#     phpize && \
+#     ./configure && \
+#     make && make install && \
+#     cd ..;
```
Comment (abnegate) [inline Dockerfile:64]: Let's remove it if we're not using it

File: tests.yaml [hunk 2]
```diff
@@ -3,40 +3,40 @@ schemaVersion: '2.0.0'
...
   - name: 'Check swoole extension'
-    path: /usr/local/lib/php/extensions/no-debug-non-zts-20200930/swoole.so
-    shouldExist: true
-  - name: 'Check swoole Debugger extension'
-    path: /usr/local/lib/php/extensions/no-debug-non-zts-20200930/yasd.so
+    path: /usr/local/lib/php/extensions/no-debug-non-zts-20220829/swoole.so
     shouldExist: true
+  # - name: 'Check swoole Debugger extension'
+  #  path: /usr/local/lib/php/extensions/no-debug-non-zts-20220829/yasd.so
+  #  shouldExist: true
```
Comment (abnegate) [inline tests.yaml:10]: Let's remove this too

**Review by abnegate — APPROVED** [review]

File: Dockerfile (hunk 1 above)
Comment (eldadfux) [reply Dockerfile]: Removed.

File: tests.yaml (hunk 2 above)
Comment (eldadfux) [reply tests.yaml]: Removed.

**Comment by eldadfux** [comment]
Removed support for Yasd as mentioned on the Yasd readme, PHP 8.1+ and Swoole 5+ support xdebug natively.

**Pull request merged**